content/engine/release-notes/25.0.md

```diff
@@ -0,0 +1,103 @@
+- Sort unconsumed build arguments before display in build output. [moby/moby#45917](https://github.com/moby/moby/pull/45917)
+- The `docker image save` tarball output is now OCI compliant. [moby/moby#44598](https://github.com/moby/moby/pull/44598)
+- The daemon no longer appends `ACCEPT` rules to the end of the `INPUT` iptables chain for encrypted overlay networks. Depending on the system's firewall configuration, a rule such as `iptables -A INPUT --dir in -p udp --dport 4789 -m policy --pol ipsec -j ACCEPT` may need to be programmed to permit incoming encrypted overlay network traffic. [moby/moby#45280](https://github.com/moby/moby/pull/45280)
+- Unpacking layers containing files with extended attributes onto a filesystem which doesn't support extended attributes will now fail. Previously, the extended attributes would be discarded if they couldn't be set, which could potentially result in containers malfunctioning. Most storage drivers already require the backing filesystem to support extended attributes so this change should have no impact on most users. The `vfs` driver is a notable exception; the previous behaviour can be restored by configuring it with `--storage-opt vfs.xattrs=i_want_broken_containers`. Enabling this storage option is highly discouraged and bug reports submitted against a daemon with this option set will be summarily closed without comment. [moby/moby#45464](https://github.com/moby/moby/pull/45464)
+- Update daemon MTU option to BridgeConfig and display warning on Windows. [moby/moby#45887](https://github.com/moby/moby/pull/45887)
+- Validate IPAM config when creating a network. Automatically fix networks created prior to this release with an --ip-range bigger than --subnet. [moby/moby#45759](https://github.com/moby/moby/pull/45759)
+- containerd image store: Add image events for `push`, `pull`, and `save` [moby/moby#46405](https://github.com/moby/moby/pull/46405)
+- containerd image store: Add support for pulling legacy schema1 images. [moby/moby#46513](https://github.com/moby/moby/pull/46513)
+- containerd image store: Add support for pushing all tags. [moby/moby#46485](https://github.com/moby/moby/pull/46485)
+- containerd image store: Add support for registry token. [moby/moby#46475](https://github.com/moby/moby/pull/46475)
+- containerd image store: Add support for showing the number of containers that use an image. [moby/moby#46511](https://github.com/moby/moby/pull/46511)
+- containerd image store: Fix `ONBUILD`, `MAINTAINER`, and healthchecks in Dockerfile instruction for support. [moby/moby#46313](https://github.com/moby/moby/pull/46313)
+- containerd image store: Fix `Pulling from` progress message. [moby/moby#46494](https://github.com/moby/moby/pull/46494)
+- containerd image store: Fix `TestInspectByPrefix` [moby/moby#46435](https://github.com/moby/moby/pull/46435)
```

Comment:
I don't think anyone cares about the `TestInspectByPrefix` integration test 😄 

We should have something like:
```suggestion
- containerd image store: Add support for referencing images via the truncated ID with `sha256:` prefix
```